extensions/everhour/src/components/TaskListItem.tsx

```diff
@@ -5,20 +5,30 @@ import { Task } from "../types";
 import { startTaskTimer, stopCurrentTaskTimer } from "../api";
 import { createResolvedToast } from "../utils";
 
+const formatSeconds = (seconds: number): string => {
+  const minutes = Math.floor(seconds / 60);
+  if (minutes >= 60) {
+    const hours = Math.floor(minutes / 60);
+    const min = minutes % 60;
+    return `${hours} ${hours === 1 ? "hour" : "hours"} and ${min} min`;
+  }
+  return `${minutes} min`;
+};
+
 export function TaskListItem({
   task,
   hasActiveTimer,
   refreshActiveTimer,
   refreshRecords,
-  todaysTimeRecords,
+  recentTimeRecords,
```

Comment:
If you add the default to an empty array here you won't have to unnecessarily pass it [here](https://github.com/raycast/extensions/pull/5023/files#diff-e43c8547c476723e027752a2e846a0ac140eb996d4fe3ec3b9176583011597c9R64).

If you chose this pattern don't forget to update the type to accept` undefined` 😉
```suggestion
  recentTimeRecords = [],
```

Reply:
ca28100 done 👍🏻